### PR TITLE
chore: add copyrights

### PR DESCRIPTION
**Pull request by christiancompton**
This PR adds copyright headers to the files that are missing it.

### CODE REVIEW

**Comment by christiancompton** [comment]
I was unsure whether to add the `lib/` dir files, as I was not 100% sure these are 'our' code.

**Comment by padamstx** [comment]
@barrett-schonefeld Could you please review this PR for me?  :)

**Comment by barrett-schonefeld** [comment]
@padamstx absolutely!

**Comment by dpopp07** [comment]
> I was unsure whether to add the lib/ dir files, as I was not 100% sure these are 'our' code.

Let me know if you saw anything specific you were looking at, but I think we want everything in `lib/` and `auth/` to have the copyright. It is all our code

File: auth/utils/file-reading-helpers.browser.ts [hunk 1]
```diff
@@ -1,3 +1,19 @@
+/**
+ * Copyright 2021 IBM Corp. All Rights Reserved.
```
Comment (dpopp07) [inline auth/utils/file-reading-helpers.browser.ts:2]: This is a little annoying but I wonder if we need to be making sure these dates are accurate. As in, the date should reflect when the file was created and the last time it was edited. Not sure how important this is but Mike always said it was. He even wrote this script to automate it - https://github.ibm.com/mkistler/lint-copyright-dates

Might be worth checking out to see if it's a simple thing to do

**Review by dpopp07 — COMMENTED** [review]
Looks good overall but we may need to look at the dates. I left a comment. Let me know if you have thoughts

**Comment by christiancompton** [comment]
@dpopp07 I have run the script on the repo to update the copyrights and updated the changes here.

**Review by barrett-schonefeld — APPROVED** [review]
Looks good!

**Review by dpopp07 — APPROVED** [review]
Looks good! 👍

**Pull request merged**